backend/pkg/database/join_tables.go

```diff
@@ -139,6 +144,12 @@ func (j *JoinTablesHandler) DeleteRelationships(params DeleteRelationshipsParams
 				return fmt.Errorf("failed to delete relationships in %q join table: %v",
 					ResourcePackagesJoinTableName, err)
 			}
+			ar := FilterIs(tx.Model(&ApplicationResources{}),
+				FieldInTable(ApplicationResourcesJoinTableName, columnJoinTableResourceID), params.ResourceIDsToRemove)
+			if err := ar.Delete(&ApplicationResources{}).Error; err != nil {
+				return fmt.Errorf("failed to delete relationships in %q join table: %v",
+					ApplicationResourcesJoinTableName, err)
+			}
```

Comment:
why it was added now?

Reply:
removing the resource_packages relationship too not just the applications_resources relations